### ISSUE TITLE
Make line numbers start higher than 1 when there is included code

### ISSUE THREAD

**Issue by presnick** [RunestoneInteractive/RunestoneComponents#237]
When there is included, hidden code in an activecode (specified in .rst file with :include: <divid>), the line numbers that are displayed with the activecode window should start at 1 + len(included_code).

This will make error messages give line numbers that make sense to users.

**Comment by bnmnetp** [comment]
Yep, this should not be too hard.  Codemirror has a configuration variable:

```
firstLineNumber (integer)
At which number to start counting lines. Default is 1.
```
We should be able to set this with the other configuration variables when we instantiate an instance of Codemirror inside the activecode component.
